README.md

```diff
@@ -47,3 +47,42 @@ myServer.run({
   })
 })
 ```
+
+## Docs :
+
+### `const server = new SQSServer()`
+
+Create a fake SQS server
+
+### `server.run(opts, cb)`
+
+ - `opts.port` ; defaults to 0
+ - `opts.host` ; defaults to `localhost`
+
+Starts the server. `cb` get's called once listening on a port.
+
+### `server.getQueue()`
+
+Returns the current array of items queued in SQS. These are shaped
+like aws SQS objects.
+
+### `server.waitForMessages(count, listener)`
+
+Get notified once N messages have been send to this fake SQS.
```

Comment:
```suggestion
Get notified once N messages have in total been sent to this fake SQS.
```

Reply:
The API was confusing to me, I thought it would call back once the next `N` messages have been sent.